### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by trojan03** [huggingface/chat-ui#312]
Hi there, 
Thanks for the great UI!
As an international user, I would like to see the UI localized in my native language. It would be great if chat-ui could detect the language of my device and automatically switch to the appropriate language. Alternatively, it would be helpful if there was an option in the settings to manually select the language.
Is this something on your roadmap?

**Comment by nsarrazin** [comment]
Hey @trojan03 ! Thanks for the question. I think it's an interesting idea but I'm not sure if it's on the roadmap. 

@gary149 any thoughts on this ?

**Comment by gary149** [comment]
Yes, I agree this is a good idea, probably as a community effort where everyone can help translate into their native language.
